eq_translations/extractable_strings.py

```diff
@@ -156,6 +156,11 @@
         "description": "Answer",
         "context": "Question",
     },
+    {
+        "json_path": "$..answers[*].instruction",
+        "description": "Checkbox Answer Instruction",
```

Comment:
The casing should be sentence case to be consistent with the other descriptions.
```suggestion
        "description": "Checkbox answer instruction",
```